sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/csv/UnivocityParser.scala

```diff
@@ -317,7 +317,15 @@ class UnivocityParser(
         if (skipRow) {
           row.setNullAt(i)
         } else {
-          row(i) = valueConverters(i).apply(getToken(tokens, i))
+          // This is required to not set value as null ,
+          // 1. If its missing value at the end of line.
+          // 2. If the missing value at the beginning of line.
```

Comment:
nit: `1. If its missing value` ... -> `1. If "the" missing value` ?

Reply:
@itholic  Have done the changes as suggested .